edx_sga/templates/staff_graded_assignment/show.html

```diff
@@ -73,8 +73,7 @@
           <td><%= assignment.timestamp %></td>
           <td>
             <% if (assignment.score !== null) { %>
-              <%= assignment.score %> /
-              <%= max_score %>
+              <%= (assignment.score / max_score) * weight %> / <%= weight %>
```

Comment:
@pdpinch @carsongee  I am show percentage out of weight.

absolute marks = (marks / total marks ) * weight 
showing <<  absolute marks / weight